### PR TITLE
Hide secrets from logs

### PR DESCRIPTION
**Pull request by saimonation**
- Move implementation from common/object to covert/format.py (tojsonstr)
- Update client/host functions to use tojsonstr
- Update UT
With this implementation, the SDK does not override the server values and displays debug messages without line breaks

### CODE REVIEW

File: cterasdk/convert/format.py [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
+
+def _to_protected_dict(o):
+    ret = copy.deepcopy(o.__dict__)
+    for key in _sdk_hidden:
+        if key in ret:
+            ret[key] = '*** The Value is Hidden by the SDK ***'
+    return ret
+
+
+def tojsonstr(obj, pretty_print=True, no_log=False):
```
Comment (ygalblum) [inline cterasdk/convert/format.py:33]: I think no_log should be True by default, no?

Comment (saimonation) [reply cterasdk/convert/format.py:33]: I'm ok with either. I thought hiding is less frequent than not hiding. We use `tojsonstr` in `show()` `show_multi()` and `show_query()` and I didn't want to make any changes to the implementation in other areas except the log prints

Comment (ygalblum) [reply cterasdk/convert/format.py:33]: I understand your point. But, I think it's cleaner to explicitly not hide details that we think should be hidden

Comment (saimonation) [reply cterasdk/convert/format.py]: @ygalblum sounds good. i've incorporated the changes

**Pull request merged**